app.js

```diff
@@ -83,17 +87,25 @@ const sessionOpts = Object.assign({
 
 app.use(session(sessionOpts));
 
+// check for cookies
+app.use(require('hof').middleware.cookies());
+
 // apps
 app.use(require('./apps/rtm/'));
 
 app.get('/cookies', (req, res) => res.render('cookies'));
 app.get('/terms-and-conditions', (req, res) => res.render('terms'));
 
+// 404's
+app.use(require('hof').middleware.notFound({
+  logger: require('./lib/logger'),
+  translate: i18n.translate.bind(i18n),
```

Comment:
it will delay execution of any next middleware until next is called, they are called synchronously afaik